### DISCUSSION TITLE
Subdomain returns a 404 error

### DISCUSSION THREAD

**Discussion by chidimmaNworah** [vercel/community#6512] [Help]
### Summary

my subdomain: shop.nailsrepublic.co does not return a page like the other domains. I get a site can't be reached page. I have done some troubleshooting like removing the DNS records and adding it back, removing the domain and adding it back, and crosschecking my domain provider to make sure it isn't expired and all this is fine but I can't figure out why the page doesn't work but when I use the Vercel provided page, it works fine

### Example

shop.nailsrepublic.co

### Steps to Reproduce

1. I set the nailsrepublic.co fine and it works but when I add the subdomain in the settings, add the ip address and everything needed, then I go ahead and to add the prefix in the project I want which is shop.nailsrepublic.com and still get a 404 page. I have even removed the entire project and brought it back but still the same issue

**Comment by amyegan** [comment]
Did you add the subdomain in project settings?

**Reply by chidimmaNworah** [reply]
Yes. I did all that. In fact, it used to work fine initially and all of a sudden stopped working. I re-did all the procedures to adding a sub domain and still it doesn't work

**Reply by amyegan** [reply]
I see `nailsrepublic.co` is associated with one of your projects, but I don't see the subdomain `shop.nailsrepublic.co` associated with any project. Can you add it on the domains tab in project settings?

**Reply by chidimmaNworah** [reply]
Yes you couldn't see it because I was still trying to troubleshoot and removed the domain. I have just added it back again and it still doesn't work. You can check now

**Reply by amyegan** [reply]
It's working correctly for me now. Do you still get a 404 error?

**Reply by chidimmaNworah** [reply]
shop.nailsrepublic.co? that's strange. It still isn't working for me. I just tried even with a completely different computer just in case it is a cache issue but still it doesn't work
![Screenshot 2024-04-16 151532](https://github.com/vercel/community/assets/78983600/89cba5c7-d3dc-4091-b382-abbe3480dd69)

**Reply by amyegan** [reply]
This is what I see
<img width="1019" alt="screenshot of site loaded correctly at subdomain address" src="https://github.com/vercel/community/assets/8247105/3b82be1e-1aaf-471f-9c05-e957f37006d9">


When we see ERR_CONNECTION_REFUSED, that's generally a local issue. Can you run this command and let me know the output?

```
curl -I https://shop.nailsrepublic.co
```

**Reply by chidimmaNworah** [reply]
This is what I get
![Screenshot 2024-04-16 160442](https://github.com/vercel/community/assets/78983600/a45a76c5-00f8-4ddf-94c4-95d549183cf9)

**Reply by chidimmaNworah** [reply]
Hello are you still with me? Why does other connected domains like https://nailsrepahop.vercel.app/ work but my subdomain doesn't? I've tried using other devices since you said it worked for you but to no avail. It is quite frustrating. I even tried to change the subdomain name to something else like buy.nailsrepublic.co but it still doesn't work but the main domain works fine. I need help with this urgently. So far, I haven't broken any rules so I need some kind of explanation

**Reply by amyegan** [reply]
This problem looks like a network or device issue, not a block by Vercel. If you're currently in Nigeria, it could be that your internet service provider blocking some Vercel sites. We've seen some recent reports of that being an issue and we're working with Nigerian ISPs to get it fixed. 

Please let me know if this is still affecting you 🙏

**Reply by chidimmaNworah** [reply]
Yes it is still affecting me. I'm positive that it isn't a personal network issue or a device issue as I have used multiple devices from different parts of the country but it still doesn't work. How soon do you think the issue with Nigerian ISPs can be resolved and also, if i host the sub domain as CNAME and domain DNS on my domain provider cpanel, which happens to be Namecheap, will it fix the issue?

**Reply by amyegan** [reply]
We have reached out to the ISPs involved, but we don't have a time estimate available. I know this is a frustrating situation. Thank you for your patience while we work to resolve the issue.

The only reliable workarounds I have found are local solutions, such as a VPN or alternative DNS server, which can only fix it on an individual basis. I have not had anyone report that updating DNS records solved the problem, but it could be worth trying. If you do try that, please let me know if it works for you. It would be good to have another option to offer others facing the same situation.
